Makefile

```diff
@@ -2,17 +2,19 @@ HOSTNAME=okta.com
 NAMESPACE=pam
 NAME=oktapam
 BINARY=terraform-provider-${NAME}
-# On verion changes, update tag-checks.yml
+# On version changes, update tag-checks.yml
 VERSION=0.2.1
 OS_ARCH=$(shell go env GOOS)_$(shell go env GOARCH)
 PLUGIN_DIR=~/.terraform.d/plugins
+INSTALL_TARGET=${PLUGIN_DIR}/${HOSTNAME}/${NAMESPACE}/${NAME}/${VERSION}/${OS_ARCH}
+TEMP_DIR=/tmp
```

Comment:
I don't think you need to copy this into tmp?

Reply:
It's being copied there as a convenient place to get it afterward. See the `ci-artifact-copy` target.

Reply:
It's likely easier to just build it in a subdirectory and mount that directory through the host.